_stylesheets/docs.css

```diff
@@ -29,7 +29,7 @@ nav#main {
     filter: drop-shadow(0.6em 0.01em 0.2em);
   }
 }
-    
+
 .navbar.navbar-default.navbar-openshift .navbar-brand.origin {
   background: url("../_images/okd_logo.svg") no-repeat scroll 0% 95%;
```

Comment:
Hi @aireilly 

I've noticed that the 4.12 equivalent file does not have the same number of lines, so maybe I could raise another PR to fix this issue in enterprise branches?

Reply:
No need. All the doc versions get built with the enterprise-4.1 version of the css for consistency.